src/app/components/Toolbar/Features/Account/ImportableAccount.tsx

```diff
@@ -18,7 +18,6 @@ export const ImportableAccount = ({
       displayCheckbox={true}
       displayAccountNumber={true}
       displayDerivation={{
-        type: account.type,
```

Comment:
![d5fd4724 oasis-wallet pages dev_create-wallet(extension375x600 + scrollbar)](https://github.com/user-attachments/assets/f7876774-6a03-4ff3-a39b-55fc746c29b7)
could increase accountsPerPage like https://github.com/oasisprotocol/oasis-wallet-web/pull/1917/files

Reply:
will keep the current value. for tablet and desktop this view is quite big.